iot-hub/Samples/device/TwinSample/TwinSample.cs

```diff
@@ -18,19 +18,19 @@ public TwinSample(DeviceClient deviceClient)
 
         public async Task RunSampleAsync()
         {
-            await _deviceClient.SetDesiredPropertyUpdateCallbackAsync(OnDesiredPropertyChangedAsync, null).ConfigureAwait(false);
+            await _deviceClient.SetDesiredPropertyUpdateCallbackAsync(OnDesiredPropertyChangedAsync, null);
 
             Console.WriteLine("Retrieving twin...");
-            Twin twin = await _deviceClient.GetTwinAsync().ConfigureAwait(false);
+            Twin twin = await _deviceClient.GetTwinAsync();
 
             Console.WriteLine("\tInitial twin value received:");
             Console.WriteLine($"\t{twin.ToJson()}");
 
             Console.WriteLine("Sending sample start time as reported property");
             TwinCollection reportedProperties = new TwinCollection();
-            reportedProperties["DateTimeLastAppLaunch"] = DateTime.Now;
+            reportedProperties["DateTimeLastAppLaunch"] = DateTime.UtcNow;

```

Comment:
Unless presenting dates to a user, they should be saved in UTC, especially when written to a global service